.editorconfig

```diff
@@ -10,9 +10,19 @@ insert_final_newline = true
 indent_style = tab
 indent_size = 4
 
-[{*.json,*kml,*.md}]
+[{.*,*.json,*.md}]
 indent_style = space
 indent_size = 2
 
+[site/includes/sitemenureplace.hbs]
```

Comment:
I wonder if this should just be inlined again

Reply:
You mean like by moving all of that file's contents into sitemenu.hbs?

Reply:
Yeah since it's a single line that is used in a single file. Maybe there was a reason for extracting it though

Reply:
Looks like @LaurentGoderre made that change in #7028.

Prior to that, ``site/layouts/default.hbs`` used to contain ``site/includes/sitemenureplace.hbs``' contents (as well as tons of other stuff that PR split into includes).

I presume ``sitemenureplace.hbs`` was just created to make ``sitemenu.hbs`` look less bloated. GCWeb doesn't reference nor override it.

I'm ok with either way. Although given that some other includes (like ``site/includes/pagetitle.hbs``) could be candidates for a similar rule in the future and since I'm juggling with 3 PRs atm, the status quo might be a better bet for the time being.